src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figures;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (firstLeg * secondLeg) / 2; //the formula of right triangle area
```

Comment:
If you want to add comment about formula please add it to all figures, or remove comment at all. I prefere second way